composer.json

```diff
@@ -18,8 +18,7 @@
     "require-dev": {
         "composer/composer": "1.0.*@dev",
         "jakub-onderka/php-parallel-lint": "~0.8",
-        "phpspec/prophecy-phpunit": "~1.0",
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "~4.8|~5.0",
```

Comment:
I think I'd rather see it as just ~4.8 so we make sure we stay compatible with 5.3.3 in the implemented tests.